pymc3/distributions/simulator.py

```diff
@@ -19,17 +19,20 @@
 
 
 class Simulator(NoDistribution):
-    def __init__(self, function, *args, **kwargs):
+    def __init__(self, function, params, *args, **kwargs):
```

Comment:
Doesn't this break existing implementations in a non-obvious way?

Maybe instead do `(self, function, **args, params=None, **kwargs)` ?